grp/simple.gi

```diff
@@ -1165,7 +1179,7 @@ local a;
   if n<168 then return 5;fi; 
   a:=Filtered(SIMPLEGPSNONL2,x->x[1]=n);
   # we have degree data up to order 2^55
-  if n<=10^55 and Length(a)=0 then
+  if n<=2^55 and Length(a)=0 then
```

Comment:
The documentation for `SimpleGroupsIterator` should also be updated. It currently says that "up to order 10^18" is supported, which clearly is outdated.

Ideally, also the documentation and code for `SmallSimpleGroup` and  `AllSmallNonabelianSimpleGroups ` should be adjusted: their documentation says the limit is 10^6, their code enforces the limit 10^18. Perhaps the limit (10^18, 10^55, 2^55, whatever it may be) could be put into a global variable or global constant, and then used from there? (well, perhaps it can't be used in the documentation this way, but at least the other places; and then a comment next to that global variable could say something like "Note: when changing this value, please also update the documentation for X, Y, Z."

@hulpke would you perhaps willing to do that? If you lack time, of course we can do it in another PR.

Reply:
I'll change documentation.

This is actually separate, fixing a stupid typo. For groups up to order 2^55 the smallest permutation degree is stored. (This is used in the library to decide whether a permutation representation is of sufficiently small degree.)